### PR TITLE
homework_2

### PR DESCRIPTION
**Pull request by junyaa**
homework_2

### CODE REVIEW

**Comment by junyaa** [comment]
fix

File: homeworks/yuriy.piddubnyak_junyaa/homework_2/task_2.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const s = 500;
+const e = 1000;
+let x;
+let i;
+
+console.log('Палиндромы от', s, 'до', e);
```
Comment (markoffden) [inline homeworks/yuriy.piddubnyak_junyaa/homework_2/task_2.js]: pls use only English lang in code

File: homeworks/yuriy.piddubnyak_junyaa/homework_2/task_1.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+let i = '##';
+const p = ' ';
+let z;
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+rl.question('Введите количество строк?\n', (x) => {
```
Comment (markoffden) [inline homeworks/yuriy.piddubnyak_junyaa/homework_2/task_1.js]: same here, only English

File: homeworks/yuriy.piddubnyak_junyaa/homework_2/task_1.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    z = x;
+    if (x == x * 1) {
```
Comment (markoffden) [inline homeworks/yuriy.piddubnyak_junyaa/homework_2/task_1.js]: try to solve task not using loose equality

**Pull request merged**